src/engine/ruleset/integrations/windows/decoders/windows-powershell-operational.yml

```diff
@@ -4,19 +4,18 @@ parents:
   - decoder/windows-event/0
 
 metadata:
-  description: Decoder for Windows Powershell Operational events
   module: Windows Powershell/Operational
   title: Decoder for Windows Powershell Operational logs
-  compatibility: >
-    This decoder has been tested on Wazuh version 4.3
+  description: Decoder for Windows Powershell Operational events
+  versions: [Vista, "7", "8", "10", "11", Server 2012, Server 2016, Server 2019, Server 2022]
+  compatibility: This decoder was tested on Windows 10, but should work on all versions of Windows Vista onwards.
   author:
     name: Wazuh, Inc.
-    date: 2023/01/23
+    date: 2023/10/10
   references:
     - https://learn.microsoft.com/en-us/powershell/scripting/windows-powershell/wmf/whats-new/script-logging?view=powershell-7.3
-  versions: [Vista, "7", "8", "10", "11", Server 2012, Server 2016, Server 2019, Server 2022]
 
-check: string_equal($windows.System.Channel.#text,Microsoft-Windows-PowerShell/Operational)
+check: $windows.System.Channel.#text == Microsoft-Windows-PowerShell/Operational
```

Comment:
We should unify the way of checking this field. I think the most legible way is how it was added in the `Windows Powershell` decoder, like this:

```
check:
  - windows.System.Channel.#text: Microsoft-Windows-PowerShell/Operational
```

What do you think?